docs/create-label-from-rate.md

```diff
@@ -1,5 +1,111 @@
 Create Label From Rate
 ======================================
-When retrieving rates for shipments using the `getRates` method, the returned information contains a `rate_id` property that can be used to purchase a label without having to refill in the shipment information repeatedly.
+[ShipEngine](www.shipengine.com) allows you programmatically create shipping labels. You can use the `create_label_from_rate` to create a shipping label from pre defined shipment information that is persisted in the `rate_id` that is passed to the function. Please see [our docs](https://www.shipengine.com/docs/labels/create-from-rate/) to learn more about creating shipping labels from a rate.
 
-> Coming Soon
+Input Parameters
+-------------------------------------
+
+The `create_label_from_rate` method requires a `rate_id` as well as some label details that are detailed in the documentation above.
+
+Output
+--------------------------------
+The `create_label_from_rate` method returns the label that was created in a response class of ShipEngine::Domain::Labels::CreateFromRate::Response.
+
+Example
+```ruby
+def create_label_from_rate_demo_function()
+	client = ShipEngine::Client.new("API-Key")
+
+	params = {
+	  validate_address: "no_validation",
+	  label_layout: "4x6",
+	  label_format: "pdf",
+	  label_download_type: "url",
+	  display_scheme: "label"
+	}
+
+	begin
+	  result = client.create_label_from_shipment_details('se-795684260', params)
```

Comment:
@j-r-j Looks like this is code for `create_label_from_shipment_details` instead of `create_label_from_rate`

Reply:
ahhh just a title mismatch 

good catch